tcms/kiwi_auth/views.py

```diff
@@ -22,6 +22,7 @@
 User = get_user_model()  # pylint: disable=invalid-name
 
 
+# pylint: disable=missing-permission-required
```

Comment:
For files which need to globally disable this checker, this line needs to go to the very top of the file, before the first import statements.

Reply:
This file still includes other class/function level disable comments so all of them need to be fixed.